tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "module": "commonjs",
     "target": "esnext",
     "esModuleInterop": true,
-    "allowJs": false
+    "allowJs": false,
+    "strictNullChecks": true
   },
-  "include": ["./src/**/*"]
+  "include": ["./src/**/**/*"]
```

Comment:
Does this change do anything? I think these two are equivalent

Reply:
@dwilliams27 typescript got mad at me initially, but I just changed it back and it looks like there are no errors, so I can update this